app/ldap.go

```diff
@@ -52,7 +52,7 @@ func (a *App) GetLdapGroup(ldapGroupID string) (*model.Group, *model.AppError) {
 		}
 	} else {
 		ae := model.NewAppError("GetLdapGroup", "ent.ldap.app_error", nil, "", http.StatusNotImplemented)
-		mlog.Error(fmt.Sprintf("%v", ae.Error()))
+		mlog.Error("LDAP group not found", mlog.String(ldapGroupID))
```

Comment:
Same here, it should be something like `mlog.Error("Unable to use ldap", mlog.Err(err))`

Reply:
UPDATED

Reply:
@vaithak The error in this case is that Ldap is not available to even do the get operation so I believe a better message would be:
 
```suggestion
		mlog.Error("Unable to use ldap", mlog.Error(ae))
```

Reply:
@marianunez what is the suggested change ? is there an updated message ?